src/gui/systray.cpp

```diff
@@ -126,7 +126,9 @@ void Systray::create()
         if (!AccountManager::instance()->accounts().isEmpty()) {
             _trayEngine->rootContext()->setContextProperty("activityModel", UserModel::instance()->currentActivityModel());
         }
-        _trayEngine->load(QStringLiteral("qrc:/qml/src/gui/tray/Window.qml"));
+
+        QQmlComponent trayWindowComponent(_trayEngine, QStringLiteral("qrc:/qml/src/gui/tray/Window.qml"));
+        _trayWindow = qobject_cast<QQuickWindow*>(trayWindowComponent.create());
```

Comment:
this `_trayWindow` will need to be eventually deleted somewhere and I am missing such code
and OK I understand that right now it will happen only at application closing

Reply:
but `QPointer` never deletes the memory it is pointing too
it will *just* be set to `nullptr` if the pointed memory is deleted by somebody else

Reply:
Okay, I changed it to a QSharedPointer